site/docs/master/httpproxy.md

```diff
@@ -252,6 +252,54 @@ The TLS **Minimum Protocol Version** a vhost should negotiate can be specified b
 - 1.2
 - 1.1 (Default)
 
+##### Fallback Certificate
+
+Contour provides virtual host based routing, so that any TLS request is routed to the appropriate service based on the hostname specified in the request's HOST header.
+As HOST Header is encrypted during TLS handshake, it can’t be used for virtual host based routing unless client sends HTTPS request specifying hostname using the SNI or the request is first decrypted using a default TLS certificate.
+Some users need to support clients which may not be using SNI.
+When an HTTPS request is received, Envoy needs to first decrypt the request using a default TLS certificate and then based on the HOST header, route it to appropriate service.
+Contour provides a way to define a default or fallback TLS certificate for an TLS request which does not provide SNI in the request.
+
+_Note: The minimum tls protocol for any fallback request is defined by the `minimumProtocolVersion` set in the Contour configuration file._
+
+###### Configuration
+
+First define the `namespace/name` in the [Contour configuration file][10] of a Kubernetes secret which will be used as the fallback certificate.
+Next configure `TLSCertificateDelegation` to specific namespaces or all (e.g. `*`) which should be allowed to enable the fallback certificate.
+Any `HTTPProxy` which enables fallback certificate delegation must have the fallback certificate delegated to the namespace in which the `HTTPProxy` object resides.
+Finally, for each root `HTTPProxy`, set the `Spec.TLS.enableFallbackCertificate` parameter to allow that fqdn to opt-in to the fallback certificate routing.
```

Comment:
Rather than "fqdn" here, suggest that we use "to allow that `HTTPProxy`" or even "allow that domain"

Reply:
Was just trying to match the language in the document. We call it a `fqdn` not `domain`, so was thinking that using the same terminology would be less confusing to users.

Reply:
Ah I see it used above in the "TLS" section. I think the change you made reads well though.